### ISSUE TITLE
For ex-mode g/pattern/p, the p is not working

### ISSUE THREAD

**Issue by Olsonist** [asvetliakov/vscode-neovim#296]
On OSX 10.15.5
with VSC 1.46.1 + Neo Vim v0.0.50
with NeoVim Path set to /usr/local/bin/nvim
and nvim --version is NVIM v0.4.3

Suppose that I search for all lines with _xyz_ in them and print them with

> :g/xyz/p

The print result shows nothing although the cursor has been moved to the last _xyz_. Also, when I am typing the p, VSC is trying to helpfully autocomplete that as packadd packloadall ...

If I delete all lines with _xyz_ in them

> :g/xyz/d

So the delete in fact works.

Basically, the print results just aren't displaying in some sort of popup.
